_posts/2024-11-02-black-python-devs-pyho-2024.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Amplifying Voices and Fostering Community: Black Python Devs as a Catalyst."
+date: 2024-11-02
+author: Kafui Alordo
+lang: en
+featured_image: /assets/images/bpd_pyho_2024.jpg
+---
+
+We’re ecstatically pleased as a community to share in the celebration of the incredible impact of Black Python Devs (BPD) on PyHo 2024 (the regional Python community conference of the people of Volta). Through an unwavering commitment to empower underrepresented/underserved developers.
+
+BPD’s dedication to serving as a safe and inclusive space for Black “Pythonistas” of all skill levels is awe inspiring. Fostering a vibrant and supportive community through providing opportunities for connection, learning, and collaboration.
+
+We appreciate Abigail Afi Gbadago, a BPD executive council member for gracing us with her presence during our #Sponsors talk session. Sharing invaluable insights into BPD’s initiatives and the opportunities for participation and collaboration within our local user group.
+
+![Black Python Devs PyHo 2024](/assets/images/bpd_pyho_2024_2.jpeg)
+
+Looking ahead, we envisage to continue our collective journey to shaping the future of Python’s open source ecosystem from a collaboration mindset over competition. BPD’s focus on growth and collaboration aligns perfectly with our shared goals and we look forward to working together to achieve even greater heights for the community both locally and globally.
+
+![Black Python Devs PyHo 2024](/assets/images/bpd_pyho_2024_3.jpg)
```

Comment:
@kafui4k - please use a descriptive alt text such as "group photo of black python devs attendees"

Reply:
updated